libs/cardano-data/src/Data/UMap.hs

```diff
@@ -11,9 +11,9 @@
 {-# LANGUAGE TypeFamilies #-}
 {-# LANGUAGE ViewPatterns #-}
 
--- | A 'UMap' (for Unified map) represents 2 'Data.Map's and one 'Data.BiMap' into a single structure.
+-- | A 'UMap' (for Unified map) represents 2 'Data.Map's into a single structure.
```

Comment:
This comment is odd.. The reference to the BiMap is only in the comment, not in code.. the definition is just this:

```data UMap coin cred pool ptr = UnifiedMap !(Map cred (Trip coin ptr pool)) !(Map ptr cred)```

I thought maybe the comment went out of sync with the code, but looking at the history, it seems to have always had this definition. So i guess the comment is just inaccurate, unless it has some other meaning that I don't see

Reply:
let's ask @TimSheard

Reply:
As I suggested during the meeting, we should change the comment to something along the lines:

```suggestion
-- | A 'UMap' (for Unified map) represents three Maps of the same size in one direction with @cred@ for 
-- keys and one more in the inverse direction with @ptr@ for keys and @cred@ for values.
```